app/build.gradle

```diff
@@ -5,6 +5,8 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'org.jetbrains.dokka'
 
+apply plugin: 'org.jetbrains.dokka'
+
```

Comment:
Why are you applying the same plugin twice?

Reply:
Thanks, I've removed the duplicate line in my latest commit.